src/AutoRest.CSharp/AutoRest.CSharp.csproj

```diff
@@ -12,11 +12,12 @@
     <SatelliteResourceLanguages>en-US</SatelliteResourceLanguages>
     <PackAsTool>true</PackAsTool>
     <RollForward>Minor</RollForward>
+    <DefineConstants>$(DefineConstants);EXPERIMENTAL</DefineConstants>
```

Comment:
Not all projects reference `Azure.Core.Experimental`, hence we need some way to exclude bits that rely on types from `Azure.Core.Experimental`. 
When do we plan to move `RequestOptions` into `Azure.Core`?